saleor/graphql/attribute/utils.py

```diff
@@ -435,9 +448,14 @@ def _validate_references(
         entity_model = cls.ENTITY_TYPE_TO_MODEL_MAPPING[
             attribute.entity_type  # type: ignore
         ]
-        ref_instances = get_nodes(references, attribute.entity_type, model=entity_model)
-        values.references = ref_instances
-        return values
+        try:
+            ref_instances = get_nodes(
+                references, attribute.entity_type, model=entity_model
+            )
+            values.references = ref_instances
+            return values
+        except GraphQLError as e:
+            raise ValidationError({ValidationError(str(e), code="graphql_error")})
```

Comment:
Did you mean (1):
```
raise ValidationError(str(e), code="graphql_error")
```
or (2)
```
raise ValidationError({"references": ValidationError(str(e), code="graphql_error")})
```
We use the 1 when we are returning a general error that isn't related to any specific input field and 2 when we are referring to a specific input field causing the error. Please also add a unit test for this error.

Reply:
Also, the code is not correct as it is not a graphql error anymore. The code should be rather `INVALID`. Don't hardcode the error code string, there is the `error_class` param that provides you with an error class to get the constant from (see other usages of `error_class` in this file).

Reply:
👍